geth/jail/extensions/console/console.go

```diff
@@ -0,0 +1,140 @@
+package console
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/jail/extensions"
+	"github.com/status-im/status-go/geth/node"
+)
+
+var (
+	// Stdout defines the default writer for the console.log
+	// delivery call.
+	Stdout io.Writer = os.Stdout
+
+	_ = extensions.Register(func(vm *otto.Otto) error {
+		return vm.Set("console", map[string]interface{}{
+			"log": consoleLog,
+		})
+	})
```

Comment:
This PR is coming from my fork, we need to move this into status itself. Will close this PR and reopen a new one for status-go.

@tiabc  @farazdagi